cmd/burrow/commands/config_options.go

```diff
@@ -10,104 +10,146 @@ import (
 )
 
 type configOptions struct {
-	validatorIndexOpt      *int
-	validatorAddressOpt    *string
-	validatorPassphraseOpt *string
-	validatorMonikerOpt    *string
-	externalAddressOpt     *string
+	configFileOpt      *string
+	genesisFileOpt     *string
+	validatorIndexOpt  *int
+	accountIndexOpt    *int
+	initAddressOpt     *string
+	initPassphraseOpt  *string
+	initMonikerOpt     *string
+	externalAddressOpt *string
 }
 
 func addConfigOptions(cmd *cli.Cmd) *configOptions {
-	spec := "[--validator-moniker=<human readable moniker>] " +
-		"[--validator-index=<index of validator in GenesisDoc> | --validator-address=<address of validator signing key>] " +
-		"[--validator-passphrase=<secret passphrase to unlock validator key>] " +
-		"[--external-address=<hostname:port>]"
+	spec := "[--moniker=<human readable moniker>] " +
+		"[--index=<index of account in GenesisDoc> " +
```

Comment:
There are still references in the docs to validator-index

Reply:
Ah sorry I should've done a grep...